package.json

```diff
@@ -38,7 +38,7 @@
 	],
 	"dependencies": {
 		"prop-types": "^15.7.2",
-		"terminal-link": "^1.2.0"
+		"terminal-link": "^2.0.0"
```

Comment:
Note: bumped `terminal-link` to 2.0.0 which changes the format of the fallback: https://github.com/sindresorhus/terminal-link/pull/12